ee/localserver/krypto-ec-middleware.go

```diff
@@ -222,6 +226,14 @@ func (e *kryptoEcMiddleware) Wrap(next http.Handler) http.Handler {
 						// so just add the session id manually
 						context.WithValue(callbackReq.Context(), multislogger.KolideSessionIdKey, kolideSessionId[0]),
 					)
+
+					// here we want to create an additional callback for presence detection, but it's only going to be valid if we have the session (saml) id
```

Comment:
I assume this is true? maybe ... I dunno, K2 could always look up the request ID